turnerlabs/yt_push
# DRACO-3865 External URL YouTube publish does not work for certain videos

Fix the streaming from HTTP to YouTube upload by ensuring multiples of 262144 bytes are sent to YouTube at a time except for the last chunk.

To test:

* Configure a YouTube media type for external URL's (drush shortcut: `drush cay --external-video`)
* Try uploading a few different external videos, you should not receive the error in the ticket description.
